sw/device/silicon_creator/lib/cert/cwt_dice_chain_entry.hjson

```diff
@@ -0,0 +1,40 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+    name: "cwt_dice_chain_entry",
+
+    variables: {
+        payload: {
+            type: "byte-array",
+            max-size: 437
+            // kCwtPayloadMaxVariableSizeBytes from bazel-bin/sw/device/silicon_creator/lib/cert/cwt_dice_chain_entry_payload.h
+        },
+        signature: {
+            type: "byte-array",
+            exact-size: 32
```

Comment:
Shoud be 64 for ECDSA-P256

Reply:
Thanks for pointing it out. It should be fixed now.